src/main/java/ru/skypro/homework/controller/AdsController.java

```diff
@@ -21,14 +22,15 @@
 @RestController
 @RequestMapping("/ads")
 public class AdsController {
+        private final AdServiceImpl adService;
```

Comment:
```suggestion
        private final AdService adService;
```
инъекцию надо делать всегда по интерфейсу, а не реализации